mantidproject/mantid
# Add read out table to workbench sliceviewer

**Description of work.**
This PR adds a table to the sliceviewer in workbench containing information about the pixel the mouse is over, in a similar way to in the spectrumviewer in mantidplot.

**To test:**
Open workbench 
Load a matrix workspace and open sliceviewer.
Move the mouse over the plot and the values should change (you can check the values by comparing with the spectrumview in mantidplot)
Swap the axes over and check the values are still correct
Unclick track cursor and the values in the table should not change unless you click the plot.

Load a MD workspace and open sliceviewer
Do the same as above. It should only display the data coordinates.

Fixes #27463

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
